RayLin.md

```diff
@@ -1,12 +1,32 @@
-# 09/26 ~ 09/30
+# 10/03 ~ 10/14
 
 - [Video Control]
   - Bug 1271765 - Visual refresh of media controls
-    - ask :astley for help with layout problem
-    - fix some mochitests and a11y tests, but still try to make them green on all platforms.
-    - a11y accessible tree is different from original XUL version. Get to tweak it in optimal way.
+    - review:
+      - Part 1: Layout reflow fix    r?dholbert
+      - Part 2: Visul refresh        r+jaws
+      - Part 3: Tests fix            r+jaws
+    - layout:
+      - discussed with :astley offline
+      - dholbert PTO til 10/17
+    - a11y:
+      - study ARIA and optimize new a11y tree
+      - get help from a11y team (:marcoZ and :surkov)
+      - ask them to verify new impl if it is still accessible
+    - l10n:
+      - don't use single quote in locale string(Bug 1294275) -> workaround with JavaScript
+    - mochitest-5:
+      - system default font breaks one of case which highly rely on precise font size(width)
+      - `font-size-adjust` fix font issue on platforms individually
+    - Hope to land by the end of next week (10/21)
+
   - Bug 1302320 - add a test to confirm that video control show "noaudio" icon when video has no audio
-    - WIP
+    - patch done and pushed. Will add review flag once Bug 1271765 landed
```

Comment:
sorry, I'll add depend soon later. 

I use 'anonid' rather than 'class' as identifier in new impl, so if this bug land before Bug 1271765, I would need to adjust the patch, but I don't want to make another review iteration of Bug 1271765.